protokube/pkg/protokube/tainter.go

```diff
@@ -68,7 +68,7 @@ func ApplyMasterTaints(kubeContext *KubernetesContext) error {
 		nodeTaintJSON := node.Annotations[v1.TaintsAnnotationKey]
 		if nodeTaintJSON != "" {
 			if nodeTaintJSON != string(taintJSON) {
-				glog.Infof("Node %q had unexpected taint: %v", node.Name, nodeTaintJSON)
+				glog.Infof("Node %q is registered with taint: %v", node.Name, nodeTaintJSON)
```

Comment:
I think that would be good (I actually think I needed it somewhere else for some reason) but not a blocker for this PR.

I would use `apply-master-taints` instead as the flag name.  But it can be separate.